packages/react-core/src/components/Modal/ModalContent.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ModalBoxBody from './ModalBoxBody';
+import ModalBoxHeader from './ModalBoxHeader';
+import ModalBoxHCloseButton from './ModalBoxCloseButton';
+import ModalBox from './ModalBox';
+import ModalBoxFooter from './ModalBoxFooter';
+import Backdrop from './Backdrop';
+import Bullseye from '../../layouts/Bullseye/Bullseye';
+
+const propTypes = {
+  /** content rendered inside the Modal. */
+  children: PropTypes.node.isRequired,
+  /** additional classes added to the button */
+  className: PropTypes.string,
+  /** Flag to show the modal */
+  show: PropTypes.bool,
+  /** Content of the Modal Header */
+  header: PropTypes.any,
+  /** Content of the Modal Footer */
+  footer: PropTypes.any,
+  /** A callback for when the close button is clicked */
+  onClose: PropTypes.func,
+  /** Creates a large version of the Modal */
+  isLarge: PropTypes.bool,
+  /* Aria label used for Modal Box Header */
+  label: PropTypes.string.isRequired
+};
+
+const defaultProps = {
+  className: '',
+  show: false,
+  header: null,
+  footer: null,
+  onClose: () => undefined,
+  isLarge: false
+};
+
+const ModalContent = ({
+  children,
+  className,
+  show,
+  header,
+  footer,
```

Comment:
@mattnolting Is is expected that the modal will only have actions/buttons in the footer, or are we expecting any kind of content here. If we are only expecting buttons I think it would be nice to see this prop as `actions` with a usage similar to 
```jsx
<Modal 
  actions={[
    <Button variant="primary" key="primary" />,
     <Button variant="secondary" key="secondary" />,
  ]}
/>
```
Then similar to the header prop we can reserce footer for custom footers.

Reply:
@dmiller9911 per @mceledonia @kybaker, `<a>` and `<span>` should be supported in the modal footer

Reply:
@dmiller9911 - I think you're suggesting that we have a default footer layout that displays actions, which can be defined on the `<Modal>` component. And if the consumer needs something custom, then the consumer can include the `<ModalBoxFooter>` for custom contents? I like this approach.

Reply:
@dmiller9911 ah, if that's what you're suggesting, I agree. Thanks for the clarification @jgiardino:)

Reply:
ah yes sorry should have probably put a larger example up:
Default
```jsx
<Modal
  actions={[<Button />]}
/>
```
Custom
```jsx
const CustomFooter = (props) => (
  <ModalBoxFooter>
    Some Custom Footer
  </ModalBoxFooter>
);
<Modal
  footer={CustomFooter}
/>
```
Another Custom option
```jsx
const CustomFooter = (propsFromModal //not sure what these are currently) => (
  <ModalBoxFooter>
    Some Custom Footer
  </ModalBoxFooter>
);
<Modal
  renderFooter={(props) => <CustomFooter {...props} />}
/>
```

Actions with Custom Footer
```jsx
const CustomFooter = (props) => (
  <ModalBoxFooter>
    {props.actions}
  </ModalBoxFooter>
);
<Modal
  renderFooter={(props) => <CustomFooter {...props} />}
/>
```

Reply:
@priley86 @karelhala opinions on propAPI for this?